README.md

```diff
@@ -0,0 +1,287 @@
+Elapsed Time:  27:45
+```
+
+In this example, hosts scxhpi1, scxsun03-s8, and scxsles9-01 are still
+building. All other hosts have completed (with time to complete). If
+a build fails, status is "Failed" (bolded).
+
+
+###Assumptions:
+
+1. For simplicity, pbuild assumes that you're using [certificates
+everywhere][]. This means that you can type "ssh
+<hostname>" from one Unix/Linux host to go directly to some other
+Unix/Linux host without username/password prompts.<br><br>
+pbuild, when starting, will verify that all required SSH keys are
+stored in your ~/.ssh/known_hosts file. If you have modified your
+configuration file, pbuild will run this verification step again to
+double check that all your hosts can be reached. This can be forced
+with --initialize.
+
+[certificates everywhere]: https://github.com/MSFTOSSMgmt/ostc-docs/blob/master/setup-sshkeys.md
+
+2. pbuild assumes that you're using 'bash' as your shell. If important,
+it can be changed to support other shells. But to date it has no such
+support.
+
+
+### Command Qualifiers to pbuild
+
+Please do ```pbuild --help``` for latest/greatest information:
+
+```
+Usage: pbuild.py [option-list] [host-list]   (use --help or -h for help)
+
+pbuild will allow you to do a build across all platforms using your own git
+clones. If a build problem occurs, all files are left completely intact,
+simplifying the steps you must perform to replicate the problem. pbuild
+supports a wide variety of options, specified below.  For more information,
+view the README file checked in with pbuild.
+
+Options:
+  -h, --help            show this help message and exit
+  --abortOnError        Immediately aborts when the first remote build fails
+                        (for future - do not use yet)
+  --attributes=TEST_ATTRS
+                        Specifies the unit test attributes that you wish to
+                        use to restrict unit tests
+  -b BRANCH, --branch=BRANCH
+                        Selects the branch for the top level project or
+                        superproject
+  -d, --debug           Build targets in DEBUG mode
+  --clone               Forces a new clone of the repository, even if
+                        repository already exists
+  --command=COMMAND     Executes the specified command string rather than
+                        buiding/running a command script to perform a remote
+                        build
+  --exclude=EXCLUDE     Overrides default exclude list from configuration file
+                        (if any); comma-separated list of hosts to exclude
+                        from the build
+  --initialize          Verify public keys in known_hosts file
+  -l, --list            List host configuration information and exit
+  --logdir=LOGDIR       Overrides 'logdir' from configuration file
+  --logdir_prior=LOGDIR_PRIOR
+                        Overrides 'logdir_prior' from configuration file
+  --nocleanup           Skip the 'tf undo' cleanup at the end of a build
```

Comment:
This is actually a stale feature, comes from pbuild.py. I removed this from the source code. I debated adding a --cleanup (to force cleanup after a successful build, currently we leave it in a built state), but decided not to unless a need arises.